custom/icds/location_reassignment/parser.py

```diff
@@ -151,3 +153,36 @@ def _validate_descendants_archived(self):
                 self.errors.append("Location %s is getting archived but the following descendants are not %s" % (
                     location.site_code, ",".join(missing_site_codes)
                 ))
+
+
+class HouseholdReassignmentParser(object):
+    def __init__(self, domain, workbook):
+        self.domain = domain
+        self.workbook = workbook
+        self.reassignments = {}  # household id mapped to new awc code
+
+    def parse(self):
+        errors = []
+        site_codes = set()
+        for worksheet in self.workbook.worksheets:
+            location_site_code = worksheet.title
+            site_codes.add(location_site_code)
+            for row in worksheet:
+                household_id = row.get(HOUSEHOLD_ID_COLUMN)
+                new_awc_code = row.get(AWC_CODE_COLUMN)
+                if not household_id:
+                    errors.append("Missing Household ID")
+                if not household_id and not new_awc_code:
+                    errors.append("Missing New AWC Code")
+                site_codes.add(new_awc_code)
```

Comment:
Shouldn't this and the line below be in an `else`?

Reply:
forgot to put continues
https://github.com/dimagi/commcare-hq/pull/27134/commits/03b8b556df774432a6e6fdba5b79609a6bdef900